src/s2train.py

```diff
@@ -30,29 +30,35 @@ def submit_to_model(detector, fmd: dict, ask_async: bool = False):
     iq_metadata = {
         "frame_num": fmd["frame_num"],
     }
+    human_review = "ALWAYS" if cloud_labeling else "NEVER"
```

Comment:
There's a third option, `DEFAULT`, which is "only when unconfident" which I think could be useful here.  So maybe this needs to be a string arg?

Reply:
Added